### ISSUE TITLE
Global breakpoints cannot be overridden

### ISSUE THREAD

**Issue by IanVS** [RafidMuhymin/astro-imagetools#79]
When setting global breakpoints in an `astro-imagetools.config.mjs` file, they always are used, even if a component defines its own set of breakpoints, those are ignored and the global breakpoints are used instead.

**Comment by SlicedSilver** [comment]
This is probably the source of the issue:
https://github.com/RafidMuhymin/astro-imagetools/blob/a7b1b479e7aa6640c0885aa657f748ccad8cfb7b/packages/astro-imagetools/api/utils/getFilteredProps.js#L88-L91

Swapping the positions of `filteredLocalProps` & `filteredGlobalConfigs` should fix the issue as the locals will override the globals:
```js
const resolvedProps = {
    ...filteredGlobalConfigs,
    ...filteredLocalProps,
  };
```

**Comment by RafidMuhymin** [comment]
A fix for this has been merged and released in `0.6.8`. Closing this issue. Feel free to reopen the issue if you are still facing any problems.
